src/main/java/com/mitchej123/hodgepodge/mixins/early/cofhcore/MixinCoFHCoreUpdateCheck.java

```diff
@@ -12,8 +14,9 @@ public class MixinCoFHCoreUpdateCheck {
      * @author mitchej123
      * @reason Update URL is long since gone
      */
-    @Overwrite(remap = false)
-    public void run() {
-        // Do nothing
+    @Inject(at = @At("HEAD"), cancellable = true, method = "run", remap = false)
+    private void hodgepodge$exit(CallbackInfo ci) {
+        ci.cancel();
```

Comment:
The idea was that any other mixin could still be applied because the instructions are all still there, so no error occurs to begin with. The mixined methods would never be called though, because of our injects at HEAD the method is immediately canceled.

Reply:
that's exactly the issue, it's not more compatible. it just doesn't throw an error on boot